docs/observable-state.md

```diff
@@ -56,92 +116,88 @@ class Doubler {
     increment() {
         this.value++
     }
+
+    *fetch() {
+        const response = yield fetch("/api/value")
+        this.value = response.json()
+    }
 }
 ```
 
+**All annotated** fields are **non-configurable**.<br>
+**All non-observable** (stateless) fields (`action`, `flow`) are **non-writable**.
+
```

Comment:
I think our tabs become a bit overwhelming now, on my browser I cant even see them all. I think it 'd be good to move subclassing section to the end of the page as separate section (below "a note on classes") and have separate details or tabs with subclass examples there.